### PR TITLE
T shanim/update readme

### PR DESCRIPTION
**Pull request by shanialbeck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,66 @@ const [templateFile, setTemplateFile] = useState<File | null>(null);
...
 ```
+### API
+`connected-workbook` exposes __WorkbookManager__, which generates a workbook with a single loaded query or table data.
+
+
+#### async `generateSingleQueryWorkbook`: `Promise<Blob>`
+
+|param   | type   | required   | description   |
+|---      |---    |---          |---            |
+|query   | [QueryInfo](#queryinfo)   | __required__  | Power Query mashup  |
```
Comment (ron-b) [inline README.md:115]: links don't work for me, at least during review, if you are sure they work, great.

File: README.md [hunk 2]
```diff
@@ -104,6 +104,66 @@ const [templateFile, setTemplateFile] = useState<File | null>(null);
...
+
+#### async `generateSingleQueryWorkbook`: `Promise<Blob>`
+
+|param   | type   | required   | description   |
+|---      |---    |---          |---            |
+|query   | [QueryInfo](#queryinfo)   | __required__  | Power Query mashup  | 
+| initialDataGrid  | [Grid](#grid)   | optional   | Initial grid data  | 
+| fileConfigs  | [FileConfigs](#fileconfigs)   | optional  | Custom file configurations |
+
+#### async `generateTableWorkbookFromHtml`: `Promise<Blob>`
```
Comment (ron-b) [inline README.md:119]: I'd stick to how we show code snippets + make sure its valid syntax, example:
```typescript
async function generateTableWorkbookFromHtml: Promise<Blob>
```

File: README.md [hunk 3]
```diff
@@ -104,6 +104,66 @@ const [templateFile, setTemplateFile] = useState<File | null>(null);
   setTemplateFile(e!.target!.files!.item(0));
 }}/>
 ```
+### API
+`connected-workbook` exposes __WorkbookManager__, which generates a workbook with a single loaded query or table data.
```
Comment (ron-b) [inline README.md:108]: The library exposes a __workbookManager__, which generates a workbook via several APIs:

**Review by ron-b — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (shanialbeck) [reply README.md:117]: I think the links work, you just can't use them when in preview mode.

**Review by ron-b — APPROVED** [review]

**Pull request merged**